README.md

```diff
@@ -11,6 +11,34 @@ This repository contains examples for various functions for DKube. Dkube support
 
 ### Examples
 
-Each pipeline example contains a pipline definition (.py file) and a config.yaml file. config.yaml shows referance values to run the pipeline. In case DKube resources needs to be created, they are specified under code, datasets and models. Pipeline arguments are specified under arguments.
+- [mnist](mnist) - this example demonstrates end to end workflow of dkube from development to deployment. 
+- [titanic](titanic) - this example demonstrates on how to use featuresets and leaderboard in DKube
+- [noteboks](notebooks) - this directory contains notebooks showing DKube API/SDK, Kubeflow Pipelines and DKube storage op usage
 
-Some values of `__xxx__` form needs to be provided by the user.
\ No newline at end of file
+#### Using script from Homedir
```

Comment:
Accessing Code from inside container

Dkube pipeline components makes all the code repos available at /mnt/dkube/home/<username>/workspace. Here is one example showing how to access  a particular script file from your code repo in dkube component.